security.rst

```diff
@@ -656,6 +656,96 @@ many other authenticators:
 
 .. _security-form-login:
 
+Login Programmatically
+----------------------
+
+.. versionadded:: 6.2
+
+    The :class:`Symfony\\Bundle\\SecurityBundle\\Security\\Security` class
+    was introduced in Symfony 6.2. In previous Symfony versions this class was
+    The :method:`Symfony\\Bundle\\SecurityBundle\\Security\\Security::login` method
+    was introduced in Symfony 6.2.
+
+Since Symfony 6.2, you can log in an user programmatically using the `login()` method of the
```

Comment:
The `Since Symfony 6.2...` can be removed, we have a `versionadded` directive before